pkg/helpers/helpers.go

```diff
@@ -100,6 +105,25 @@ func DecodeSnowflakeID(id string) sdk.ObjectIdentifier {
 	}
 }
 
+func DecodeSnowflakeParameterID(id string) sdk.ObjectIdentifier {
+	parts := strings.Split(id, ParameterIDDelimiter)
```

Comment:
just FYI this could be problematic if you have "." in your name. I.e.

> "test.db"."myschema"."mystream" 


This is a perfectly valid identifier, but when split would result in 4 parts. The correct thing to do is to split on double quotes, if there are double quotes.

Reply:
Yeah, but it's also not that easy, because we should enforce that if the user wants to use quotes, they should use them for each part of the id (e.g. "db".schema."table" should not be accepted).

This is a much broader topic than just fixing it in this PR. Let's not aim to solve it perfectly here and instead let's dedicate a separate task to discuss and handle it correctly.

Reply:
I'll fix it and write tests for it, so it will be clear that we wouldn't get any unintentional behaviour

Reply:
And please add a parametrized test for such a method.